common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -250,26 +247,34 @@
             "error"
         ],
         "@typescript-eslint/no-unsafe-assignment": [
-            "error"
+            "warn"
         ],
         "@typescript-eslint/no-unsafe-call": [
-            "error"
+            "warn"
         ],
         "@typescript-eslint/no-unsafe-member-access": [
-            "error"
+            "warn"
         ],
         "@typescript-eslint/no-unsafe-return": [
             "error"
         ],
         "@typescript-eslint/no-unused-expressions": [
-            "error"
+            "warn"
         ],
         "@typescript-eslint/no-unused-vars": [
             "warn"
         ],
         "@typescript-eslint/no-var-requires": [
             "error"
         ],
+        "@typescript-eslint/object-curly-spacing": [
+            "error",
+            "always",
+            {
+                "arraysInObjects": false,
+                "objectsInObjects": false
+            }
```

Comment:
I think we should remove these two exceptions for object-curly-spacing (i.e. just have error/always)

Reply:
Will do.

Reply:
Fixed in latest.